appveyor.yml

```diff
@@ -8,21 +8,24 @@ environment:
 
   matrix:
 
-    - TOOLCHAIN: "ninja-vs-15-2017-win64-cxx17"
-      PROJECT_DIR: examples\catkin
-      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+    # FIXME: ninja, nmake and vs can't find python package catkin, which could point to some
```

Comment:
Python looks fine:
```
-- Found PythonInterp: C:/Python34-x64/python.exe (found version "3.4.4")
-- Using PYTHON_EXECUTABLE: C:/Python34-x64/python.exe
-- Using default Python package layout
-- Found PY_em: C:\Python34-x64\lib\site-packages\em.py
-- Using empy: C:/Python34-x64/lib/site-packages/em.py
```

The only error I see is:
```
ninja: error: mkdir(CMakeFiles/catkin_topological_order_executable_install.dir/catkin_generated/add_python_executable/catkin_topological_order_executable_install): No such file or directory
```

Probably that's the cause:
```
C:/projects/hunter/_testing/Hunter/_Base/d60c692/212f99a/b889b11/Build/catkin/Build/catkin-Release-prefix/src/catkin-Release-build/CMakeFiles/catkin_topological_order_executable_devel.dir/
  has 188 characters.  The maximum full path to an object file is 250
  characters (see CMAKE_OBJECT_PATH_MAX).  Object file
    catkin_generated/add_python_executable/catkin_topological_order_executable_devel/catkin_topological_order.cpp.obj
  cannot be safely placed under this directory.  The build may not work
  correctly.
CMake Warning
```

Can you try:
- https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#fix-appveyor-path-too-long-error

?

Reply:
> Can you try:
> 
>     * https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#fix-appveyor-path-too-long-error
> 
> 
> ?

This seems to have fixed it. My builds are still ongoing but most of the toolchains passed already. I will test with all the toolchains and update the pull request.

Reply:
I think this is ready for review again.